guides/common/modules/ref_webhooks-templates.adoc

```diff
@@ -0,0 +1,6 @@
+[id="webhooks-templates_{context}"]
+= Templates
+
+Webhook HTTP payload must be created via {Project} template syntax. The subject of the event is available via `@object` variable. {Project} ships with an example template for every single event available. For more information, see {ManagingHostsDocURL}appe-Red_Hat_Satellite-Managing_Hosts-Template_Writing_Reference[Template Writing Reference] in _Managing Hosts_ and available template macros and methods, visit `/templates_doc` on a {ProjectServer}.
```

Comment:
Is the link to Managing Hosts here correct? It looks very strange.

Reply:
Please build locally and test @ianf77 
It is best practice.